client/src/nonblocking.rs

```diff
@@ -0,0 +1,188 @@
+use crate::{
+    parse_logs_response, ClientError, EventContext, Program, ProgramAccountsIterator,
+    RequestBuilder,
+};
+use anchor_lang::{prelude::Pubkey, AccountDeserialize, Discriminator};
+use futures::stream::StreamExt;
+use futures::Future;
+use solana_account_decoder::UiAccountEncoding;
+use solana_client::{
+    nonblocking::{pubsub_client::PubsubClient, rpc_client::RpcClient},
+    pubsub_client::PubsubClientError,
```

Comment:
This errors on v1.14.18. We should instead use [`non_blocking::pubsub_client::PubsubClientError`](https://docs.rs/solana-client/1.14.18/solana_client/nonblocking/pubsub_client/enum.PubsubClientError.html).

Reply:
I will fix it

Reply:
I will channge by the new error of the v1.16